lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -419,13 +448,37 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
 
     private loadProperties(node: Node, loadBasicProps = true, loadGroupedProps = true, loadTags = true, loadCategories = true) {
         if (node) {
+            const requests = {};
+
             if (loadBasicProps) {
-                this.basicProperties$ = this.getProperties(node);
+                requests['properties'] = this.getProperties(node);
             }
+
             if (loadGroupedProps) {
-                this.groupedProperties$ = this.contentMetadataService.getGroupedProperties(node, this.preset);
+                requests['groupedProperties'] = this.contentMetadataService.getGroupedProperties(node, this.preset);
+            }
+
+            if (this.displayPredictions) {
+                requests['predictions'] = this.loadPredictionsForNode(this.node.id);
             }
 
+            forkJoin(requests).subscribe(({ predictions, properties, groupedProperties }: ({ predictions: Prediction[]; properties: CardViewItem[]; groupedProperties: CardViewGroup[] })) => {
```

Comment:
Same here, this subscription also needs unsubscribing

Reply:
hmm, I think forkJoin automatically cleans up subscriptions

Reply:
Yes you are right, forkJoin handles the cleanup. You can skip this comment then. Thanks